### ISSUE TITLE
Add graphviz as dependency so pip could install it automatically

### ISSUE THREAD

**Issue by Petrox** [inventree/inventree-wireviz#37]
This would solve the docker container customization with every new release overhead for any users here.

automatic apt-getting in the container env is decided to be not implemented
https://github.com/inventree/InvenTree/issues/5718

**Comment by SchrodingersGat** [comment]
Can this be installed via pip?

**Comment by Petrox** [comment]
```
pip3 install graphviz
Collecting graphviz
  Using cached https://files.pythonhosted.org/packages/9d/fb/886e8ec7862989afc0c35d15813b6c665fe134cc6027cdde2fa300abe9a2/graphviz-0.19.1-py3-none-any.whl
Installing collected packages: graphviz
Successfully installed graphviz-0.19.1

```

**Comment by SchrodingersGat** [comment]
Right, but the python `graphviz` package is already installed as part of [wireviz](https://github.com/wireviz/WireViz/blob/a16f2e47573644149caaef0c420ecf07ef9f1dd1/setup.py#L27)

Is this not getting installed automatically?

**Comment by Petrox** [comment]
TBH I didn't check if it installs it or not, I've read the readme some time ago that it depends on graphviz and I should install it.

> WireWiz requires GraphViz to be installed in order to work. See the [GraphViz download page](https://graphviz.org/download/) for OS-specific instructions.

Maybe it is not necessary at all if graphviz (python package) is installed?

When I was using the stable container I got the error, that it could not find the graphviz binary, so it either not included in the pip package or not in PATH inside the container.

**Comment by SchrodingersGat** [comment]
To be clear, the graphviz python package gets installed automatically, as part of this plugin, but you need to have the `graphviz` binary also installed on your system - I don't think  PIP takes care of this part
